### PR TITLE
K8SPG-145 PG db 1.1.0 release preps

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Review by tplavcic — APPROVED** [review]

File: charts/pg-db/README.md [hunk 1]
```diff
@@ -68,12 +68,12 @@ The chart can be customized using the following configurable parameters:
 | `pgPrimary.expose.serviceType`                     | K8S service type for the PostgreSQL deployment | `ClusterIP`                              |
 | `pmm.enabled` | Enable integration with [Percona Monitoring and Management software](https://www.percona.com/blog/2020/07/23/using-percona-kubernetes-operators-with-percona-monitoring-and-management/) | `false` |
 | `pmm.image.repository`              | PMM Container image repository                                           | `percona/pmm-client` |
-| `pmm.image.tag`                     | PMM Container image tag                                       | `2.18.0`                              |
+| `pmm.image.tag`                     | PMM Container image tag                                       | `2.21.0`                              |
```
Comment (hors) [inline charts/pg-db/README.md:71]: Maybe we need to use 2.24.0 or 2.23.0

File: charts/pg-db/README.md [hunk 2]
```diff
@@ -108,13 +108,17 @@ The chart can be customized using the following configurable parameters:
...
-| `secrets.postgres`                     | postges user password (superuser, not accessible via pgbouncer)                                       |`autogrenerated by operator`|
-| `secrets.pgbouncer`                     | pgbouncer user password                                        |`autogrenerated by operator`|
-| `secrets.<default_user>`                     | Default user password                                        |`autogrenerated by operator`|
+| `pgBadger.enabled`        | Switch on pgBadger                                                            | `false` |
+| `pgBadger.image`          | pgBadger image                                                                | `percona/percona-postgresql-operator:1.1.0-ppg14-pgbadger` |
+| `pgBadger.port`           | pgBadger port                                                                 | `10000` |
+| `secrets.name`            | Database secrets object name. Object will be autogenerated if the name is not explicitly specified       |`<cluster_name>-users-secrets`|
```
Comment (hors) [inline charts/pg-db/README.md:114]:
```suggestion
| `secrets.name`            | Database secrets object name. Object will be autogenerated if the name is not explicitly specified       |`<cluster_name>-users`|
```

File: charts/pg-db/templates/user-secret.yaml [hunk 3]
```diff
@@ -1,17 +1,18 @@
-{{- range $key, $value := .Values.secrets }}
-{{- if $value }}
+{{- if and (( or (( .Values.secrets.primaryuser )) (( .Values.secrets.postgres )) (( .Values.secrets.pgbouncer )) (( .Values.secrets.pguser )))) (( not .Values.secrets.name)) }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "pg-database.fullname" $ }}-{{ $key }}-secret
+  name: {{ include "pg-database.fullname" $ }}-users-secret
```
Comment (hors) [inline charts/pg-db/templates/user-secret.yaml:6]:
```suggestion
  name: {{ include "pg-database.fullname" $ }}-users
```

**Review by hors — CHANGES_REQUESTED** [review]

File: charts/pg-db/README.md (hunk 2 above)
Comment (cap1984) [reply charts/pg-db/README.md]: done

File: charts/pg-db/README.md (hunk 1 above)
Comment (cap1984) [reply charts/pg-db/README.md]: done

File: charts/pg-db/templates/user-secret.yaml (hunk 3 above)
Comment (cap1984) [reply charts/pg-db/templates/user-secret.yaml]: done

**Review by hors — APPROVED** [review]

**Pull request merged**